package/boot/uboot-envtools/files/ramips

```diff
@@ -30,6 +30,19 @@ ampedwireless,ally-00x19k|\
 ampedwireless,ally-r1900k)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x20000" "4"
 	;;
+beeline,smartbox-flash|\
```

Comment:
I would prefer if you do not move this block and only add the new entry in alphabetical order.

Reply:
Thanks. Block was moved back and just new entry added.

Reply:
@hauke, thanks. Block was moved back and just new entry added.